package.json

```diff
@@ -64,7 +65,7 @@
     "root-check": "^1.0.0",
     "sort-on": "^3.0.0",
     "string-length": "^2.0.0",
-    "tabtab": "^1.3.2",
+    "tabtab": "^2.2.2",
```

Comment:
As soon as it's ready (probably this weekend), it'll need to be bumped to `^3.0.0`

Reply:
Okay, we'll hold off merging until then.

Reply:
Sounds good to me 👍 I want to take a little bit more time. I won't be able to work much more this weekend, and I want to take time with Wilson to coordinate our efforts. Thanks @SBoudrias